projects/plugins/jetpack/_inc/client/my-plan/style.scss

```diff
@@ -3,63 +3,64 @@
 @import '../scss/typography';
 
 .jp-landing__plans {
-  	margin-top: 48px;
+	margin-top: 48px;
 	margin-bottom: 24px;
-  	box-shadow: 0 0 40px rgba(0, 0, 0, 0.08);
+	box-shadow: 0 0 40px rgba(0, 0, 0, 0.08);
 
 	.dops-button {
 		margin-right: 10px;
 	}
 
 	.dops-card.is-compact {
-	  margin-bottom: 0;
-	  border-bottom: 1px solid var(--jp-gray);
+		margin-bottom: 0;
+		border-bottom: 1px solid var(--jp-gray);
 
-	  	@include breakpoint( "<960px" ) {
-		border-bottom: 0;
-	  	}
+		@include breakpoint( "<960px" ) {
+			border-bottom: 0;
+		}
 	}
 
-  	.dops-card:first-child {
-	  border-radius: 4px 4px 0 0;
+	.dops-card:first-child {
+		border-radius: 4px 4px 0 0;
 	}
 
-  	.dops-card:last-child {
-	  border-radius: 0 0 4px 4px;
+	.dops-card:last-child {
+		border-radius: 0 0 4px 4px;
 
-	  @include breakpoint( "<960px" ) {
-		padding-top: 0;
-	  }
+		@include breakpoint( "<960px" ) {
+			padding-top: 0;
+		}
 	}
 
-  	.dops-card:nth-last-child(2) {
-	  @include breakpoint( "<960px" ) {
-		border-bottom: 0;
-	  }
+	.dops-card:nth-last-child(2) {
+		@include breakpoint( "<960px" ) {
+			border-bottom: 0;
+		}
 	}
 
-  	.my-plan-card__tag-line a {
-	  color: $black;
-	  text-decoration-line: underline;
+	.my-plan-card__tag-line a,
+	a.jp-landing__licensing-actions-link {
```

Comment:
This is the only real change in the file. The rest is just switching from spaces to tabs.